src/components/ErrorView.js

```diff
@@ -15,7 +14,6 @@ export const styles = {
   },
   jsonFrame: {
     padding: '0.5rem',
-    backgroundColor: colors.light(),
```

Comment:
Since useThemeFromContext is a hook, it needs to be used in a component. Is there any better way of moving backgroundColor out of style and into inline.

Reply:
my thought would be to change styles here to be a function that accepts the colors object as an argument and returns the desired style props object..

Reply:
Oooh, I like that. Let me see if I can put something together.

Reply:
Oh nice, this was my only feedback and it looks like you're already on top of it